code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -87,62 +86,87 @@
 	throw_message = "does nothing to the tough hide of the"
 	pre_attack_icon = "goliath2"
 	crusher_loot = /obj/item/crusher_trophy/goliath_tentacle
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/goliath = 2, /obj/item/stack/sheet/animalhide/goliath_hide = 1, /obj/item/stack/sheet/bone = 2)
 	loot = list()
 	stat_attack = UNCONSCIOUS
 	robust_searching = 1
 
-/mob/living/simple_animal/hostile/asteroid/goliath/make_shiny()
-	name = "precursor goliath"
-	real_name = name
-	desc = "Due to their stone hide, goliaths are biologically immortal, although future generations evolved to look much different. This goliath is likely a very early ancestor to many others here, and at least several centuries old."
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/random/Initialize()
+	. = ..()
+	if(prob(1))
+		new /mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient(loc)
+		return INITIALIZE_HINT_QDEL
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient
+	name = "ancient goliath"
+	desc = "Goliaths are biologically immortal, and rare specimens have survived for centuries. This one is clearly ancient, and its tentacles constantly churn the earth around it."
 	icon_state = "Goliath"
 	icon_living = "Goliath"
 	icon_aggro = "Goliath_alert"
 	icon_dead = "Goliath_dead"
+	maxHealth = 400
+	health = 400
+	speed = 4
 	pre_attack_icon = "Goliath_preattack"
 	throw_message = "does nothing to the rocky hide of the"
 	loot = list(/obj/item/stack/sheet/animalhide/goliath_hide) //A throwback to the asteroid days
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/goliath = 2, /obj/item/stack/sheet/bone = 2)
+	crusher_drop_mod = 75
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient/Life()
+	. = ..()
+	if(isturf(loc))
+		for(var/turf/open/T in orange(4, loc))
```

Comment:
>```orange()``` every life tick

cache it, and the turf it was taken from (eg: the loc at the time), and only update it:
- every 4 life ticks
or
- the goliath moves a set distance from the turf the original was taken from.

Reply:
you can't escape the orange